inlong-sort/sort-flink/sort-flink-v1.13/sort-connectors/iceberg/pom.xml

```diff
@@ -143,8 +143,8 @@
                                     <shadedPattern>org.apache.inlong.sort.iceberg.shaded.com.amazonaws</shadedPattern>
                                 </relocation>
                                 <relocation>
-                                    <pattern>org.apache.inlong.sort.base</pattern>
-                                    <shadedPattern>org.apache.inlong.sort.iceberg.shaded.org.apache.inlong.sort.base</shadedPattern>
+                                    <pattern>org.apache.inlong.sort</pattern>
+                                    <shadedPattern>org.apache.inlong.sort.iceberg.shaded.org.apache.inlong.sort</shadedPattern>
```

Comment:
@lordcheng10 <pattern>org.apache.inlong.sort</pattern> and <pattern>org.apache.inlong.sort.base</pattern> will be effected each other?

Reply:
Fixed

Reply:
Fixed